content/en/blog/2021/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: 2021 Posts
+description: Blog posts for 2021.
+weight: 7
+icon: blog
+decoration: dot
+list_by_publishdate: true
+---
```

Comment:
Someone's already added this, so you'll need to remove it

Reply:
Updated. Thanks!